test/stress/fill_flush.c

```diff
@@ -0,0 +1,73 @@
+#include "test/jemalloc_test.h"
+#include "test/bench.h"
+
+#define SMALL_ALLOC_SIZE 128
+#define LARGE_ALLOC_SIZE SC_LARGE_MINCLASS
+#define NALLOCS 1000
+
+void *volatile allocs[NALLOCS];
```

Comment:
Without volatile, the alloc-then-free in a loop has an unfair advantage; the compiler can leave the allocation in a register (not doing the store-then-load).

(I guess there's still an unfair difference, since filling a load out of a store buffer is different than filling it out of cache, but I expect that to at least be closer).

I'll add a comment.